mvrx/build.gradle

```diff
@@ -12,11 +12,7 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 27
```

Comment:
Since you are updating everything, you could also change this to 28.

Reply:
For the sdk it does not make sense to have an artificially high target SDK. As it might pull up the garget SDK requirements for implementers without any need.